tests/agents/microchain/test_functions.py

```diff
@@ -108,3 +109,10 @@ def test_engine_help(market_type: MarketType) -> None:
         engine.register(function(market_type=market_type))
 
     print(engine.help)
+
+
+@pytest.mark.parametrize("market_type", [MarketType.OMEN])
+def test_get_probability(market_type: MarketType) -> None:
+    market_id = "0x0020d13c89140b47e10db54cbd53852b90bc1391"
+    get_market_probability = GetMarketProbability(market_type=market_type)
+    get_market_probability(market_id)
```

Comment:
Is the assertion missing here?

Reply:
Yes good point. I noticed that the way we calculate p_yes for closed omen markets is broken atm, so I made this task https://github.com/gnosis/prediction-market-agent-tooling/issues/181

I've added an assertion here that will fail once that task is resolved, but I think that's fine.